src/commands/addons/addons.mts

```diff
@@ -1,32 +1,72 @@
- 
-import { createAddonsAuthCommand } from './addons-auth.mjs'
-import { createAddonsConfigCommand } from './addons-config.mjs'
-import { createAddonsCreateCommand } from './addons-create.mjs'
-import { createAddonsDeleteCommand } from './addons-delete.mjs'
-import { createAddonsListCommand } from './addons-list.mjs'
-
-/**
- * The addons command
- * @param {import('commander').OptionValues} options
- * @param {import('../base-command.mjs').default} command
- */
-// @ts-expect-error TS(7006) FIXME: Parameter 'options' implicitly has an 'any' type.
-const addons = (options, command) => {
+import { OptionValues } from 'commander'
+
+import BaseCommand from '../base-command.mjs'
+
+const addons = (options: OptionValues, command: BaseCommand) => {
   command.help()
 }
 
-/**
- * Creates the `netlify addons` command
- * @param {import('../base-command.mjs').default} program
- * @returns
- */
-// @ts-expect-error TS(7006) FIXME: Parameter 'program' implicitly has an 'any' type.
-export const createAddonsCommand = (program) => {
-  createAddonsAuthCommand(program)
-  createAddonsConfigCommand(program)
-  createAddonsCreateCommand(program)
-  createAddonsDeleteCommand(program)
-  createAddonsListCommand(program)
+export const createAddonsCommand = (program: BaseCommand) => {
+  program
+  .command('addons:auth', { hidden: true })
+  .alias('addon:auth')
+  .argument('<name>', 'Add-on slug')
+  .description('Login to add-on provider')
+  .action(async (addonName: string, options: OptionValues, command: BaseCommand) => {
+    const { addonsAuth } = await import('./addons-auth.mjs')
+    await addonsAuth(addonName, options, command)
+  })
+
+  program
+  .command('addons:config', { hidden: true })
+  .alias('addon:config')
+  .argument('<name>', 'Add-on namespace')
+  .description('Configure add-on settings')
+  // allow for any flags. Handy for variadic configuration options
+  .allowUnknownOption(true)
+  .action(async (addonName: string, options: OptionValues, command: BaseCommand) => {
+    const { addonsConfig } = await import('./addons-config.mjs')
+    await addonsConfig(addonName, options, command)
+  })
+
+  program
+  .command('addons:create', { hidden: true })
+  .alias('addon:create')
+  .argument('<name>', 'Add-on namespace')
+  .description(
+    `Add an add-on extension to your site
+Add-ons are a way to extend the functionality of your Netlify site`,
+  )
+  // allow for any flags. Handy for variadic configuration options
+  .allowUnknownOption(true)
+  .action(async (addonName: string, options: OptionValues, command: BaseCommand) => {
+    const { addonsCreate } = await import('./addons-create.mjs')
+    await addonsCreate(addonName, options, command)
+  })
+
+  program
+  .command('addons:delete', { hidden: true })
```

Comment:
The indentation looks a bit off here. I don't think we've changed the Prettier settings to run on `.mts` files?

Reply:
Ah yeah @ericapisani is this in the plans with what you're doing now?

Reply:
I believe we need to add the new extensions here: https://github.com/netlify/cli/blob/7ab484bb94e0eed680a6e17e1eb34e5aaea9fe8d/package.json#L79.

Reply:
Added config in https://github.com/netlify/cli/pull/6187, and ran format. Should be prettier now :D